modules/database/src/ioc/db/dbAccess.c

```diff
@@ -1101,6 +1104,17 @@ static long dbPutFieldLink(DBADDR *paddr,
     if (status)
         return status;
 
+    /* do not apply per-file default from parser for changes after iocInit() */
```

Comment:
I think this described behavior is right, we ran a test at APS just before Christmas where I redirected an internal link in our main timing IOC to an external IOC. I don't think that's an unusual thing to want to do.

Reply:
I should note that even it will always be possible to retarget a link as CA.  Applying the per-file default would only required the user making the change know to append " REM" (or " EXT").

I chose not to required this as it seems to me like more annoyance than added value.  An interactive change will anyway provide immediate feedback.  (eg.  caput will show a new link value with "CA")